2025-05-09 01:56:37,403 - DEBUG    - Valid bulb creation passed     - ( test_db_integrity.py : 72 )
